webv2/web.go

```diff
@@ -2984,17 +2988,19 @@ type GeneratedResources struct {
 	BucketName   string `json:"BucketName"`
 	BucketUrl    string `json:"BucketUrl"`
 	//Used for single instance migration flow
-	DataStreamJobName       string `json:"DataStreamJobName"`
-	DataStreamJobUrl        string `json:"DataStreamJobUrl"`
-	DataflowJobName         string `json:"DataflowJobName"`
-	DataflowJobUrl          string `json:"DataflowJobUrl"`
-	DataflowGcloudCmd       string `json:"DataflowGcloudCmd"`
-	PubsubTopicName         string `json:"PubsubTopicName"`
-	PubsubTopicUrl          string `json:"PubsubTopicUrl"`
-	PubsubSubscriptionName  string `json:"PubsubSubscriptionName"`
-	PubsubSubscriptionUrl   string `json:"PubsubSubscriptionUrl"`
-	MonitoringDashboardName string `json:"MonitoringDashboardName"`
-	MonitoringDashboardUrl  string `json:"MonitoringDashboardUrl"`
+	DataStreamJobName          string `json:"DataStreamJobName"`
+	DataStreamJobUrl           string `json:"DataStreamJobUrl"`
+	DataflowJobName            string `json:"DataflowJobName"`
+	DataflowJobUrl             string `json:"DataflowJobUrl"`
+	DataflowGcloudCmd          string `json:"DataflowGcloudCmd"`
+	PubsubTopicName            string `json:"PubsubTopicName"`
+	PubsubTopicUrl             string `json:"PubsubTopicUrl"`
+	PubsubSubscriptionName     string `json:"PubsubSubscriptionName"`
+	PubsubSubscriptionUrl      string `json:"PubsubSubscriptionUrl"`
+	MonitoringDashboardName    string `json:"MonitoringDashboardName"`
+	MonitoringDashboardUrl     string `json:"MonitoringDashboardUrl"`
+	AggMonitoringDashboardName string `json:"AggMonitoringDashboardName"`
+	AggMonitoringDashboardUrl  string `json:"AggMonitoringDashboardUrl"`
```

Comment:
This should be done via a struct for `MonitoringResources`.

Reply:
I am not sure I understand this comment. Can you elaborate?